meta/ee-bindep.txt

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+# List your controller-side system package requirements here if exist
+# to support easy execution environments building for your collection users
+# https://ansible.readthedocs.io/en/latest/getting_started_ee/index.html
+# If there are no such dependencies, please just remove the file.
```

Comment:
Actually I would not recommend to delete these files, to avoid `bindep.txt` or `requirements.txt` from collection root being used by ansible-builder if these are not intended as EE deps.

Reply:
Oh, great point! Replaced the line with another one advising not to delete the files even if no dependencies. PTAL